src/Universe.cpp

```diff
@@ -893,7 +877,7 @@ Universe* Lattice::getUniverse(int lat_x, int lat_y) const {
  * @brief Return a 2D vector of the Universes in the Lattice.
  * @return 2D vector of Universes
  */
-std::vector<std::vector<std::pair<int, Universe*>>>
+std::vector< std::vector<std::vector<std::pair<int, Universe*>>>>
```

Comment:
The Google C++ style guide does not have one correct way to format this line. To summarize for an example template `std::vector<std::vector<int>>`:
- C++11 is okay with `>>` so `std::vector<std::vector<int>>` is fine.
- C++03 requires a space between the ending angle brackets `> >` so at least `std::vector<std::vector<int> >` is required.
- Google C++ style guide is okay with either of these or symmetric spacing like `std::vector< std::vector<int> >`.

The precedent in OpenMOC looks to be symmetric spacing, so I'm going to change to that.